geode-core/src/main/java/org/apache/geode/internal/concurrent/CompactConcurrentHashSet2.java

```diff
@@ -353,8 +353,8 @@ public String toString() {
     public boolean equals(Object o) {
       Object k, u;
```

Comment:
It looks like you can delete these variables

Reply:
done